test/fixtures/encapsulation.out.css

```diff
@@ -0,0 +1,90 @@
+.Component,
+.Encapsulation {
+  border-collapse: separate;
+  border-spacing: 0;
+  caption-side: top;
+  color: initial;
+  cursor: auto;
+  direction: ltr;
+  empty-cells: show;
+  font-size-adjust: none;
+  font-family: initial;
+  font-size: medium;
+  font-style: normal;
+  font-stretch: normal;
+  font-variant: normal;
+  font-weight: normal;
+  font: initial;
+  letter-spacing: normal;
+  line-height: normal;
+  list-style-image: none;
+  list-style-position: outside;
+  list-style-type: disc;
+  list-style: initial;
+  orphans: 2;
+  quotes: initial;
+  tab-size: 8;
+  text-align: initial;
+  text-align-last: auto;
+  -webkit-text-decoration-color: initial;
+          text-decoration-color: initial;
+  text-indent: 0;
+  text-justify: auto;
+  text-shadow: none;
+  text-transform: none;
+  visibility: visible;
+  white-space: normal;
+  widows: 2;
+  word-break: normal;
+  word-spacing: normal;
+  word-wrap: normal;
+}
+.Component,
+.Component-item,
+.Component-item,
```

Comment:
I believe that this comes from 

```css
.Component-item { /* ... */ }

@media (--media-query) {
  .Component-item { /* ... */ }
}
```
Maybe we can open an issue and push a branch with a failing test?
Not a blocker for us anyway.